driver/src/value.rs

```diff
@@ -12,9 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use arrow::datatypes::i256;
```

Comment:
this should be tagged with `#[cfg(feature = "flight-sql")]`

Reply:
NumberValue need this.